js/navigation.js

```diff
@@ -25,12 +25,12 @@
 		return;
 	}
 
-	if ( -1 === menu.className.indexOf( 'nav-menu' ) ) {
+	if ( menu.classList.contains( 'nav-menu' ) ) {
```

Comment:
We have to add the `!` (logical NOT) operator here to match the old implementation. We want to check if the `menu` don't contain the `nav-menu` class and not the contrary.

```suggestion
	if ( ! menu.classList.contains( 'nav-menu' ) ) {
```

Reply:
You're right, I don't know where my head was...

Reply:
it's okay, it happens to the best of us especially in these troubled times.